### PR TITLE
ref(tests) Move a few more tests to RTL

### PR DESCRIPTION
**Pull request by markstory**
Move a few more tests to RTL.

### CODE REVIEW

File: tests/js/spec/components/group/inboxReason.spec.jsx [hunk 1]
```diff
@@ -44,7 +49,12 @@ describe('InboxReason', () => {
...
     );
-    const tooltip = mountWithTheme(wrapper.find('Tooltip').prop('title'));
-    expect(tooltip.text()).toContain('Affected 10 user(s)');
+    const tag = screen.getByText('Unignored');
+    userEvent.hover(tag);
+
+    // Text is split up because of translations
+    expect(await screen.findByText('Affected')).toBeInTheDocument();
+    expect(await screen.findByText('10')).toBeInTheDocument();
+    expect(await screen.findByText('user(s)')).toBeInTheDocument();
```
Comment (scttcper) [inline tests/js/spec/components/group/inboxReason.spec.jsx:58]: these probably don't all need to be awaits, but its also a little gross to check for all 3

**Review by scttcper — APPROVED** [review]
nice

**Review by priscilawebdev — APPROVED** [review]
left 1 suggestion but it looks good to me

File: tests/js/spec/components/group/inboxReason.spec.jsx (hunk 1 above)
Comment (markstory) [reply tests/js/spec/components/group/inboxReason.spec.jsx:58]: oops, that is copy paste derpiness.

**Pull request merged**